components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1826,6 +1829,28 @@ private void translateInformation(Element root) throws FormatException {
         }
       }
 
+      Element sNode = getFirstNode(dimensions, "S");
+      if (sNode != null) {
+        NodeList scenes = sNode.getElementsByTagName("Scene");
+        int nextPosition = 0;
+        for (int i=0; i<scenes.getLength(); i++) {
+          Element scene = (Element) scenes.item(i);
+          NodeList positions = scene.getElementsByTagName("Position");
+          for (int p=0; p<positions.getLength(); p++) {
+            Element position = (Element) positions.item(p);
+            String x = position.getAttribute("X");
+            String y = position.getAttribute("Y");
+            String z = position.getAttribute("Z");
+            if (nextPosition < positionsX.length && positionsX[nextPosition] == null) {
+              positionsX[nextPosition] = new Length(DataTools.parseDouble(x), UNITS.MICROM);
+              positionsY[nextPosition] = new Length(DataTools.parseDouble(y), UNITS.MICROM);
+              positionsZ[nextPosition] = new Length(DataTools.parseDouble(z), UNITS.MICROM);
+              nextPosition++;
```

Comment:
There are 2 failures for the units in https://ci.openmicroscopy.org/view/Failing/job/BIOFORMATS-DEV-merge-full-repository/577

PlanePositions: FAILED (X position unit series 0, plane 0 (expected reference frame, actual µm)) for 
curated/zeiss-czi/nottingham/incorrect-colors/SpinalChord63x_1 SIMcrop_Structured Illumination.czi and curated/zeiss-czi/nottingham/incorrect-colors/Image 6_Structured Illumination.czi

Looking at the files in Zen Im seeing a stage position of X: -3.28 mm and Y: -2.63 mm for SpinalChord and X: 13.45 mm Y: -3.52 mm for Image6

Checking some of the other stage values from Zen against the new config PR and the updated configs look to be correct. Did these 2 files simply miss the config update for some reason?